packages/formation/js/index.js

```diff
@@ -1,11 +1,14 @@
 import loadAccordionHandler from './accordion';
+import SideBarMenu from './sidenav';
```

Comment:
Once you've made the below changes in the sidenav.js file, you should import `addSidenavListeners` and call that where you're calling `SideBarMenu` now instead.